website/docs/guides/using-directives.md

```diff
@@ -12,25 +12,25 @@ The `using` directives mechanism lets you define configuration information withi
 `using` directives are basically key-value pairs that let you provide multiple values to a single key. For instance, this command:
 
 ```scala
```

Comment:
This whole section needs to be reworded, since it speaks about 3 ways and now we have only 2. So it should default to comments nad mention top level annotation syntax

Reply:
We should also mention there what can be valid key and how nesting works